site/docs/4.15.0/overview/releaseNotes.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Apache BookKeeper 4.15.0 Release Notes
+---
+
+Release 4.15 addresses multiple CVEs, adds more configuration options, extends REST API, 
```

Comment:
I would say:
"Release 4.15 addresses multiple CVEs" -> "includes many upgrades to third party libraries marked with CVEs"

Reply:
fixed